src/ui/windows/TogglDesktop/TogglDesktop/ui/Resources/DesignUpdate/Buttons.xaml

```diff
@@ -358,9 +358,28 @@
                                    RadiusX="15"
                                    RadiusY="7">
                         </Rectangle>
-                        <Viewbox Stretch="None">
-                            <Path Opacity="0.348" Fill="{DynamicResource Toggl.DarkGrayBrush}" Data="M 0 5h4.25v4.25h1.5v-4.25h4.25v-1.5h-4.25v-4.25h-1.5v4.25h-4.25Z"/>
-                        </Viewbox>
+                        <ContentControl Content="{StaticResource Toggl.Timeline.AddNewEntry}"/>
+                    </Grid>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+    <Style TargetType="Button" x:Key="Toggl.Timeline.GapButtonOpen">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type Button}">
+                    <Grid UseLayoutRounding="False" Background="Transparent">
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto"/>
+                            <RowDefinition Height="*"/>
+                        </Grid.RowDefinitions>
+                        <Path SnapsToDevicePixels="True" Stroke="{DynamicResource Toggl.LightGrayBrush}" StrokeThickness="2" StrokeDashArray="1 1" Grid.Row="0"
+                              Data="M 0,10 A 9,9 0 1 1 20,10">

```

Comment:
Good idea to draw the arc separately!
Arc gets cut on the edges though. I've played with it a bit, so I suggest to make it go from 1,10 to 19,10, and set StrokeDashArray to "0.94 0.94" to keep the symmetry.
![image](https://user-images.githubusercontent.com/16451391/100652705-fcb3b200-334f-11eb-8795-096cb3eb1a2e.png)